src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/KeywordsTestCases.java

```diff
@@ -36,5 +34,24 @@ public void getKeywordsUsingText() {
 		}
 		assertTrue(keywordFound);
 	}
+	
+	private KeywordsRequest buildTestRequest() {
+		KeywordsRequest kr = new KeywordsRequest();
+		String paragraph = "The polymer is made by a company called Qenos from Australian gas extracted"
+				+ " from the huge fields of the Cooper and Surat Basins in the arid lands out near the"
+				+ " Queensland, New South Wales and South Australian borders.";
+		kr.setSource(paragraph);
+		kr.setShowSourceText(false);
+		kr.setCquery(null);
+		kr.setKnowledgeGraph(null);
+		kr.setMaxRetrieve(null);
+		kr.setSentiment(null);
+		kr.setXpath(null);
+		kr.setSourceText(null);
```

Comment:
There are no need to put all this values to null, just set the parameters that have value and all the others parameters will be null, it would be even better if you create a constructor for that.